src/Type/Schema.php

```diff
@@ -99,6 +99,10 @@ public function __construct($config)
         $this->extensionASTNodes = $config->extensionASTNodes;
 
         $this->config = $config;
+
+        foreach ($this->config->types as $type) {
```

Comment:
Users may specify `types` in a multitude of ways, its type is `iterable<Type&NamedType>|(callable(): iterable<Type&NamedType>)|iterable<(callable(): Type&NamedType)>|(callable(): iterable<(callable(): Type&NamedType)>)`. The code you wrote does not work with some of those options.